api-report/container-runtime.api.md

```diff
@@ -346,7 +346,7 @@ export interface IEnqueueSummarizeOptions extends IOnDemandSummarizeOptions {
 
 // @public
 export interface IGarbageCollectionRuntime {
-    closeFn(error?: ICriticalContainerError): void;
+    closeFn: (error?: ICriticalContainerError) => void;
```

Comment:
@agarwal-navin why is this exported?

Reply:
I mean why is `IGarbageCollectionRuntime` exported.  (I had to change how closeFn was written to work around an incorrect lint error detection)